src/java.base/share/classes/javax/net/ssl/SSLParameters.java

```diff
@@ -696,4 +697,76 @@ public void setApplicationProtocols(String[] protocols) {
         }
         applicationProtocols = tempProtocols;
     }
+
+    /**
+     * Returns a prioritized array of signature scheme names that can be used
+     * over the SSL/TLS/DTLS protocols.
+     * <p>
+     * The returned array includes {@code String} names from the list of
+     * standard signature schemes in the <a href=
+     * "{@docRoot}/../specs/security/standard-names.html#signature-schemes">
+     * Signature Schemes</a> section of the Java Cryptography
+     * Architecture Standard Algorithm Name Documentation, and may also
```

Comment:
The correct name is "Java Security Standard Algorithm Names Specification". Same comment below for `setSignatureSchemes`.

Reply:
Good catch!